packages/cli/src/oclif/commands/register.js

```diff
@@ -32,7 +138,42 @@ RegisterCommand.args = [
       "Your integrations's public title. Asked interactively if not present.",
   },
 ];
-RegisterCommand.flags = buildFlags();
+
+RegisterCommand.flags = buildFlags({
+  commandFlags: {
+    desc: flags.string({
+      char: 'D',
+      description:
+        'A sentence describing your app in 140 characters or less, e.g. "Trello is a team collaboration tool to organize tasks and keep projects on track."',
+    }),
+    url: flags.string({
+      char: 'u',
+      description:
+        'The homepage URL of your app. This will also be used as the email domain that would allow team members to join your integration as a Collaborator.',
+    }),
+    audience: flags.string({
+      char: 'a',
+      description:
+        'Are you building a public or private integration? Public integration development receives priority support and launch promotion from the Zapier team.',
+    }),
+    role: flags.string({
+      char: 'r',
+      description:
+        "What is your relationship with the app you're integrating with Zapier?",
+    }),
+    category: flags.string({
+      char: 'c',
+      description:
+        "How would you categorize your app? Choose the most appropriate option for your app's core features.",
+    }),
+    subscribe: flags.boolean({
+      char: 's',
+      description:
+        'Get tips and recommendations about this integration along with our monthly newsletter that details the performance of your integration and the latest Zapier news.',
+      allowNo: true,
+    }),
+  },
+});
 RegisterCommand.examples = [
   'zapier register',
   'zapier register "My Cool Integration"',
```

Comment:
Can we add some more examples here?

Reply:
@eliangcs I added more here - 286afb7d0bd75538601a69b5baa102bdb32c908f. Do you think these are thorough enough?